orienteer-core/src/main/java/org/orienteer/core/component/table/OrienteerDataTable.java

```diff
@@ -87,6 +82,11 @@ public OrienteerDataTable(String id, List<? extends IColumn<T, S>> columns,
 		add(UpdateOnActionPerformedEventBehavior.INSTANCE_ALL_CONTINUE);
 	}
 
+	@Override
+	public void setTableFilterForm(FilterForm<IODataFilter<T, S>> filterForm) {
```

Comment:
Method is "set", but internally it's adding new toolbar. If "set" will be invoked twice it will lead to double toolbar, but it's not expected behavior from "set" methods.